inc/fields.class.php

```diff
@@ -107,7 +108,7 @@ public static function getNames() {
       // Get localized names of field types
       foreach (PluginFormcreatorFields::getClasses() as $field_type => $classname) {
          $classname = self::getFieldClassname($field_type);
-         if ($classname == PluginFormcreatorTagField::class && !$plugin->isActivated('tag')) {
+         if ($classname == UndefinedField::class) {
```

Comment:
exclude undefined field from available types. Leave tags field available.

We may want to maintain tag and fields types unavailable, if we add a new method in PluginFormcreatorFirldInterface, with a slight difference with isPrerequisite(). one for dependencies out of the plugin, the other for some kind of validation of question settings.